documentation.html

```diff
@@ -1114,11 +1120,554 @@ <h4 class="component-heading">Lists</h4>
+            class="code-example"
+            sandbox="allow-scripts allow-same-origin"
+            loading="lazy"
+          >
+          </iframe>
+        </section>
+        <!--  Navigation Section End  -->
+
+        <!--  Slider Section Start   -->
+        <section class="component-section">
+          <h4 class="component-heading">Slider</h4>
+          <p class="component-sub-heading">
+            Slider or Range are used for selecting the range. Like range of
+            sound in video apps, range of price in e-commerce apps etc. For
+            using slider just copy our html and replace the icon by your
+            required one.
+          </p>
+          <p class="component-sub-heading-bold">Example of Slider with icon</p>
+
+          <div class="component-example flex-column flex-gap-1">
+            <div class="slider-container">
+              <i class="fa fa-volume-up" aria-hidden="true"></i>
+              <input type="range" min="10" max="100" value="15" />
+            </div>
+          </div>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28253%2C242%2C242%2C0.96%29&t=cobalt&wt=none&l=htmlmixed&width=893.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Cdiv%2520class%2520%253D%2520%2522slider-container%2522%253E%250A%2520%2520%253Ci%2520class%253D%2522fa%2520fa-volume-up%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%253Cinput%2520type%2520%253D%2520%2522range%2522%2520min%2520%253D%2520%252210%2522%2520max%2520%253D%2520%2522100%2522%2520value%2520%253D%2520%252215%2522%253E%253C%252Finput%253E%250A%253C%252Fdiv%253E"
+            class="code-example"
+            sandbox="allow-scripts allow-same-origin"
+            loading="lazy"
+          >
+          </iframe>
+        </section>
+        <!--  Slider Section End   -->
+
+        <!--  Forms Section Start   -->
+        <section class="component-section">
+          <h4 class="component-heading">Forms</h4>
+          <p class="component-sub-heading">
+            Forms are used to take the information from the user.
+          </p>
+          <p class="component-sub-heading-bold">Example of Forms</p>
+          <div
+            class="component-example flex-row flex-row-center-horizontal-vertical"
+          >
+            <form class="authentication-form">
+              <h2
+                class="font-large weight-semi-bold authentication-form-heading"
+              >
+                <span class="logo-text font-large">FiT</span>BiT
+              </h2>
+              <p
+                class="font-medium-large weight-semi-bold authentication-form-sub-heading"
+              >
+                LOGIN INTO YOUR ACCOUNT
```

Comment:
just  a suggestion but Login is more preferred these days as it is concise and delivers the same meaning to the user

Reply:
Done